docs/src/tutorials/supported_operations.md

```diff
@@ -29,6 +30,13 @@ As per specification the configuration file needs to be a `toml` file which desc
 
 `thin-edge.io` stores the operations configuration files in the `/etc/tedge/operations/<cloud-provider>/` directory.
 
+
+## `thin-edge.io child` Supported Operations
+
+When a new child device is added, the boot-straping of child device will create `/etc/tedge/operations/<cloud-provider>/<child-device>` directory to store the supported operations by that particular child device.
+It creates a file per Supported Operation. The operation can be dynamically added and removed. A special file placed in `/etc/tedge/operations/<cloud-provider>/<child-device>` directory
+will indicate that an operation is supported by that `child device`.
```

Comment:
While writing markdown docs, it's a good practice to keep each sentence on its own line. Have re-ordered the sentences as well:
```suggestion
Every file placed in the `/etc/tedge/operations/<cloud-provider>/<child-device>` directory
indicates an operation supported by that child device.
The operation files can be dynamically added and removed.
```

Reply:
Fixed